### PR TITLE
Update to Index Exchange amp-ad tags to reduce dependency on AMP core

### PR DESCRIPTION
**Pull request by ixpaciga**
Support functions and some of the logic in ix() itself is being migrated into a new external library to reduce dependency on pull requests into the AMP core itself.

### CODE REVIEW

**Comment by ixpaciga** [comment]
@lannka - could you let us know when we can expect comments? Thanks.

**Comment by ixpaciga** [comment]
@zhouyx - can you provide an estimate when you'll be able to look at this?

File: ads/ix.md [hunk 1]
```diff
@@ -16,22 +16,9 @@ limitations under the License.
...
 
-Index supports both direct ad tags and Header Tag style bidding using Doubleclick as the ad server.
+Index supports Header Tag style bidding using Doubleclick as the ad server.
 
-## Examples
-
-### Ad tag ###
-
-```html
-<amp-ad width=300 height=250
```
Comment (zhouyx) [inline ads/ix.md]: to make sure, the direct ad tags are not supported. If this is no longer a valid "ix" ad, please also remove it from the ads.amp.html examples.

**Comment by ixpaciga** [comment]
@zhouyx - thanks for the comment, I've updated the ads examples to be consistent with the readme.

**Comment by ixpaciga** [comment]
@zhouyx - can you provide an ETA for further comments?

File: ads/ix.js [hunk 2]
```diff
@@ -49,52 +49,18 @@ export function ix(global, data) {
...
-    }
-
-    if (typeof global._IndexRequestData === 'undefined') {
-      global._IndexRequestData = {};
-      global._IndexRequestData.impIDToSlotID = {};
-      global._IndexRequestData.reqOptions = {};
-      global._IndexRequestData.targetIDToBid = {};
     }
 
+    global._IndexRequestData = undefined;
```
Comment (zhouyx) [inline ads/ix.js]: what's this for?

**Comment by zhouyx** [comment]
@ixpaciga LGTM. One last comment.

File: ads/ix.js (hunk 2 above)
Comment (ixpaciga) [inline ads/ix.js]: We need it to exist in the global object passed to loadScript, though we may remove this in a future refactoring.

**Comment by ixpaciga** [comment]
@zhouyx I addressed your comment above, are further changes required?

File: ads/ix.js (hunk 2 above)
Comment (zhouyx) [inline ads/ix.js]: But setting it to `undefined` doesn't do anything.
By default ` global._IndexRequestData` will return undefined.

**Comment by ixpaciga** [comment]
@zhouyx - I don't believe the Travis failure is related to our change. Can you confirm or rebuild?

**Comment by zhouyx** [comment]
@ixpaciga LGTM. Thanks for the Pull Request. Merged.

**Pull request merged**